tasks.py

```diff
@@ -23,7 +23,7 @@
 if sys.platform == 'win32':
     DIST_DIR = os.environ['localappdata'] + '\\GOG.com\\Galaxy\\plugins\\installed'
     PLATFORM = "win32"
-    PYTHON_EXE = "python.exe"
+    PYTHON_EXE = "py"
```

Comment:
Can't merge this, afk for likely the rest of the day, but i can look via phone. Is it possible to check if py is installed? Otherwise, we'd need to use regular `python` 

Also, using py over python means we need to specify version (py -3.7 iirc).
This technically is an issue you introduced, but it's our fault because we assumed one python installation. Py is the better choice because it works with multiple installs of python, but we need to check if it's in the path first. I'll see if i can find a way to check if py is installed

Reply:
I found a SO article about that: https://stackoverflow.com/questions/27967795/how-do-i-check-if-an-application-is-installed-in-windows-using-python

Shutil which for "py". If true, set it to "py -3.7" or py -37 whichever it is. If not, set to python.

Reply:
I think I've done it, not sure how Python handles NULLs/empty/blankStrings though..?

I tested the which() function and it gives the following on my laptop:

>>> shutil.which("py")
'C:\\Windows\\py.EXE'
>>> shutil.which("python")
'.\\python.EXE'
>>> shutil.which("pyqwerty")
>>>

Reply:
It looks like either null or the empty string. Running an if statement on either would return false, so just `if shutil.which("py"):<set variable to py with version>` should work. I'll check the commit. I think i have line commit permission so i can just fix it myself if it's wrong. Ngl though, i just got back home and am exhausted so I'll probably take a nap and check it later this evening.